modules/rosa-cluster-autoscaler-cli.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-autoscale-cli-during_{context}"]
+= Enable autoscaling during cluster creation with the ROSA CLI
+
+You can use the ROSA CLI (`rosa`) to set cluster-wide autoscaling behavior during cluster creation.
+
+.Procedure
+
+. During cluster creation, type `--enable autoscaling` after the cluster name.
+
+.Example:
+[source,terminal]
+----
+$ rosa create cluster --cluster-name <cluster_name> --enable-autoscaling
```

Comment:
If we only specify `--enable-autoscaling`, it will only enable machineautoscaler. Maybe need to add a note,  at least one parameter needs to be specified to enable clusterautoscaler.

Reply:
@aldiazRH I would remove this example if it doesn't do what we want, that is to enable cluster autoscaling.